crates/sui-types/src/digests.rs

```diff
@@ -280,6 +280,16 @@ impl fmt::UpperHex for CheckpointContentsDigest {
     }
 }
 
+/// A digest of a cerificate, which commits to the signatures as well as the tx.
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct CertificateDigest(Sha3Digest);
+
+impl CertificateDigest {
+    pub const fn new(digest: [u8; 32]) -> Self {
+        Self(Sha3Digest::new(digest))
```

Comment:
As discussed in another thread, we could gradually replace sha3 from internal structs with blake2b (which is generally faster) - and we use blake2b already in NW. Not required for this PR, this will be another SHA3 replacement PR

Reply:
Do we want to do this for on-chain data? e.g. TransactionDigest, CheckpointDigest, etc? Or is blake2 not considered safe enough for that?

Reply:
Yes for anything, especially long objects (where blake2b has an obvious advantage). But in another PR, @jonas-lj is on that, he already ran some benchmarks (amortized, about 3x faster in Macbooks):
![hash](https://user-images.githubusercontent.com/25659162/224180612-106f5dd1-0e11-43fb-8d8d-3b9ed1f45b62.png)